tests/test_format_version_conversion.py

```diff
@@ -39,3 +43,27 @@ def test_model_format_version_conversion(unet2d_nuclei_broad_before_latest, unet
     for key, item in actual.items():
         assert key in expected
         assert item == expected[key]
+
+
+# todo: break forward compatibility on major version difference?
+@pytest.mark.parametrize("v_diff", [(0, 0, 1), (0, 1, 0), (1, 0, 0), (0, 1, 1)])
+def test_forward_compatible(v_diff: Tuple[int, int, int], unet2d_nuclei_broad_latest):
+    from bioimageio.spec import load_raw_resource_description
+    from bioimageio.spec.model import format_version
+
+    fv_key = "format_version"
+
+    model_data = yaml.load(unet2d_nuclei_broad_latest)
+
+    v_latest: Version = Version(format_version)
+    v_future: str = ".".join(
```

Comment:
maybe it's good to look at this at the end of a workday without any stamina - why is minor always incremented irrespective of v_diff?

Reply:
I removed it. was left from an implementation draft. good catch!